sdk/Sdk.Generators/FunctionExecutor/FunctionExecutorGenerator.Parser.cs

```diff
@@ -28,7 +28,7 @@ internal ICollection<ExecutableFunction> GetFunctions(IEnumerable<IMethodSymbol>
 
                 foreach (IMethodSymbol method in methods.Where(m=>m.DeclaredAccessibility == Accessibility.Public))
                 {
-                    _context.CancellationToken.ThrowIfCancellationRequested();
+                    _context.CancellationToken.ThrowIfCancellationRequested();  // Why is this using the invocations CancellationToken?
```

Comment:
This is the source generator cancellation token, not functions invocation (ie: cancel source generation itself, not cancellation consumption in the generated code). This usage is good.

Reply:
I should have double checked the context of my blind search xD